src/tests/nurse/test_views.py

```diff
@@ -162,36 +162,38 @@ def test_endpoint(self):
         assert self.url == "/prescription/1/send-email/"
 
     def test_send_email_to_doctor(self, client, prescription):
-        response = client.post(self.url)
+        response = client.post(self.url, {"additional_info": "Additional information"})
```

Comment:
DRY: Maybe save an instance of a valid payload as a TestSendEmailToDoctorView class attribute so next time we add more field or modify it we don't have to do it x times for each test

Reply:
I refactored